### PR TITLE
koa: Fix vary field parameter

### PR DESCRIPTION
**Pull request by hustLer2k**
Vary can accept either a single string or an array of strings, as per vary's [docs](https://github.com/jshttp/vary#varyres-field) and [types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1b85e933696518a58ca05d48c31065e0d20430ca/types/vary/index.d.ts#L10)

Please fill in this template.

- ✔️ Use a meaningful title for the pull request. Include the name of the package modified.
- ✔️ Test the change in your own code. (Compile and run.)
- ✔️ [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- ✔️ Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- ✔️ Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- ✔️ [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:
If changing an existing definition:
- ✔️ Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1b85e933696518a58ca05d48c31065e0d20430ca/types/vary/index.d.ts#L10
- ✔️ If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/koa/index.d.ts [hunk 1]
```diff
@@ -442,7 +442,7 @@ declare interface ContextDelegatedResponse {
 
 declare class Application<
     StateT = Application.DefaultState,
-    ContextT = Application.DefaultContext
+    ContextT = Application.DefaultContext,
```
Comment (pkuczynski) [inline types/koa/index.d.ts:445]: Such changes are irrelevant to the subject of this PR and should be reverted

File: types/koa/index.d.ts [hunk 2]
```diff
@@ -333,7 +333,7 @@ declare interface ContextDelegatedResponse {
     /**
      * Vary on `field`.
      */
-    vary(field: string): void;
```
Comment (pkuczynski) [inline types/koa/index.d.ts:336]: This change seems valid: https://github.com/jshttp/vary/blob/5d725d059b3871025cf753e9dfa08924d0bcfa8f/index.js#L129

File: types/koa/test/index.ts [hunk 3]
```diff
@@ -62,6 +62,9 @@ app.use(ctx => {
     ctx.acceptsLanguages(['']); // $ExpectType string | false
     ctx.is(''); // $ExpectType string | false | null
     ctx.is(['']); // $ExpectType string | false | null
+
+    ctx.vary("Origin");
```
Comment (pkuczynski) [inline types/koa/test/index.ts:66]: You should add `// $ExpectType`

**Review by pkuczynski — CHANGES_REQUESTED** [review]

**Comment by hustLer2k** [comment]
@pkuczynski

**Review by pkuczynski — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]
@hustLer2k thanks!

**Comment by hustLer2k** [comment]
ready to merge boss

**Pull request merged**